subsys/bluetooth/controller/ll_sw/nordic/lll/lll_prof.c

```diff
@@ -267,6 +287,22 @@ static int send(struct node_rx_pdu *rx)
 	return 0;
 }
 
+static uint16_t latency_get(void)
+{
+	uint16_t latency;
+
+	/* calculate the elapsed time in us since on-air radio packet end
+	 * to ISR entry
+	 */
+#if defined(HAL_RADIO_GPIO_HAVE_PA_PIN)
+	latency = timestamp_latency - timestamp_radio_end;
+#else /* !HAL_RADIO_GPIO_HAVE_PA_PIN */
+	latency = timestamp_latency - radio_tmr_end_get();
+#endif /* !HAL_RADIO_GPIO_HAVE_PA_PIN */
```

Comment:
Possible to use IS_ENABLED?

Reply:
one changed... the other has `timestamp_radio_end` variable conditionally defined.